src/main/java/uk/gov/hmcts/reform/sendletter/controllers/ReportsController.java

```diff
@@ -28,24 +31,18 @@ public ReportsController(ReportsService reportsService) {
         this.reportsService = reportsService;
     }
 
-    @GetMapping(path = "/count-summary", produces = MediaType.APPLICATION_JSON_VALUE)
+    @GetMapping(path = "/count-summary", produces = MediaType.APPLICATION_OCTET_STREAM_VALUE)
     @ApiOperation("Retrieves uploaded letters count summary report")
-    public LettersCountSummaryResponse getCountSummary(
+    public ResponseEntity getCountSummary(
         @RequestParam(name = "date") @DateTimeFormat(iso = DATE) LocalDate date
-    ) {
+    ) throws IOException {
```

Comment:
Should this exception be handled independently and then custom one included in controller advice?